static/docs/commands-reference/remote:add.md

```diff
@@ -57,8 +65,7 @@ specific settings in your config, that you don't want to track and share through
 
 * `-d`, `-default` - commands like `dvc pull`, `dvc push`, `dvc fetch` will be
 using this remote by default to save or retrieve data files unless `-r` option
-is specified for them. Use `dvc config` to unset/change the default remote:
```

Comment:
I think it's fine to mention this in two places if it's confusing. No need to remove it from here. Different ppl read in a different way.